src/test/java/InternSearch/pages/SearchPageSearch.java

```diff
@@ -0,0 +1,79 @@
+package InternSearch.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class SearchPageSearch extends BasePageSearch {
+
+    //Locators
+    @FindBy(css = "[type='search']")
+    private WebElement searchInput;
+
+    @FindBy(xpath = "//*[@data-testid='song-excerpts']//button[@data-test='view-all-songs-btn']")
+    private WebElement songresult;
+    //locator for correct song
+    @FindBy(xpath = "//*[@id='songResultsWrapper']//td[@class='title']")
+    private WebElement isThatCorrectSong;
+    //locator is correct artist
+    @FindBy(xpath = "//*[@id='songResultsWrapper']//td[@class='artist']")
+    private WebElement isThatCorrectartist;
+
+    //locator for correct Album
+    @FindBy(xpath = "//*[@id='songResultsWrapper']//td[@class='album']")
+    private WebElement isThatCorrectAlbum;
+    //search result-search area
+    @FindBy(css = "#searchExcerptsWrapper h1")
+    private WebElement searchResult;
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a “Code Smell”

Reply:
I deleted the empty lines.